ministryofjustice/parliamentary-questions
# Ct 3699 switch smoketest and other on staging

## Description

The related JIRA ticket has more information for the background and context
 
This PR aims to tidy up the cron jobs  running on different envs, what I did are below
- Switched off SmokeTest
- Remove run-pg.sh entirely as 
    . those steps are relies on environments (IMPORT_URL,  DOCKER_STATE , APP_MASTER_SERVER etc.) which I couldn't see anywhere except CREATE_USER_FROM_TEST
    .Migration step has been replaced with migration_job
    .reseeding data should be only on non-Prod env, but couldn't see such check on this file
- Remove the job of sanitising email and creating test users from staging env
- Change the job of trimming data to trim the unassigned cases only in order to keep the testings easier on staging env

## Self-review checklist


### Screenshots


### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3699
### Deployment


### Manual testing instructions
